packages/editor/src/client/routes/polls/pollList.tsx

```diff
@@ -0,0 +1,200 @@
+import {ApolloError} from '@apollo/client'
+import {PlayOutline} from '@rsuite/icons'
+import TrashIcon from '@rsuite/icons/legacy/Trash'
+import OffIcon from '@rsuite/icons/Off'
+import WaitIcon from '@rsuite/icons/Wait'
+import React, {useEffect, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Link} from 'react-router-dom'
+import {FlexboxGrid, IconButton, Message, Pagination, Table, toaster} from 'rsuite'
+
+import {FullPoll, usePollsQuery} from '../../api'
+import {CreatePollBtn} from '../../atoms/poll/createPollBtn'
+import {DeletePollModal} from '../../atoms/poll/deletePollModal'
+import {dateTimeLocalString, DEFAULT_MAX_TABLE_PAGES, DEFAULT_TABLE_PAGE_SIZES} from '../../utility'
+
+export function PollList() {
+  const {t} = useTranslation()
+  const [pollDelete, setPollDelete] = useState<FullPoll | undefined>(undefined)
+  const [page, setPage] = useState<number>(1)
+  const [limit, setLimit] = useState<number>(10)
+
+  /**
+   * Handling error on loading polls.
+   */
+  const onErrorToast = (error: ApolloError) => {
+    if (error?.message) {
+      toaster.push(
+        <Message type="error" showIcon closable duration={3000}>
+          {error?.message}
+        </Message>
+      )
+    }
+  }
+
+  const {data, loading, refetch} = usePollsQuery({
+    fetchPolicy: 'no-cache',
+    variables: {
+      take: limit,
+      skip: (page - 1) * limit
+    },
+    onError: onErrorToast
+  })
+
+  /**
+   * Refetch data
+   */
+  useEffect(() => {
+    refetch({
+      take: limit,
+      skip: (page - 1) * limit
+    })
+  }, [page, limit])
+
+  /**
+   * UI HELPERS
+   */
+  function pollStateView(poll: FullPoll) {
+    const now = new Date()
+    const closedAt = poll.closedAt ? new Date(poll.closedAt) : undefined
+
+    // poll has been closed
+    if (closedAt && now.getTime() >= closedAt.getTime()) {
+      return (
+        <>
```

Comment:
Fragments in this function are not needed

Reply:
Yep. And added a tooltip too.